docs-new/docs/contribute/general/refinement-guide.md

```diff
@@ -0,0 +1,52 @@
+# Refinement Timeblock
+
+During the Refinement Timeblock, maintainers engage in focused technical
+discussions on open issues and pull requests.
+This dedicated time allows for in-depth conversations, knowledge sharing,
+and collective decision-making.
+It is an opportunity for the team to synchronize their understanding of
+ongoing developments, address challenges, and ensure a common vision for the project.
+
+**Purpose and Goals:**
+
+- **Alignment:** Ensure a shared understanding among maintainers regarding ongoing
+developments, project goals, and upcoming tickets.
+- **Technical Depth:** Delve into the technical aspects of open issues and pull
+requests, facilitating a deeper understanding of proposed Pull requests.
+- **Feedback Loop:** Establish an effective feedback loop where maintainers can
+provide insights and guidance on contributions.
+- **Decision-Making:** Make collective decisions on the adoption of new features,
+changes, and improvements based on technical merit.
+
+This process aligns with our commitment to open-source principles, ensuring that
+technical discussions are inclusive, transparent, and beneficial for the
+entire Keptn community.
+
+## Contributor guidance
+
+Contributors proposing new features are encouraged to participate in
+[refinement meetings](https://community.cncf.io/keptn-community/) related
+to their contributions.
+This provides valuable insights into ongoing technical discussions and
+aligns their efforts with the broader project vision.
+Follow these steps:
+
+1. Review the project's [contributing guide](../docs/contrib-guidelines-docs.md)
```

Comment:
If you use `1.` for each list item, the renderer should take care of doing the 1, 2, 3, etc stuff.  Makes it nice when you add or delete an item so you don't have to manully renumber everything.